### ISSUE TITLE
Syntax Highlighting breaks for blame view when some line of a chain call is commented

### ISSUE THREAD

**Issue by eldiaboloz** [go-gitea/gitea#6895]
- Gitea version (or commit ref): 1.9.0+dev-217-g91c2c237f
- Git version: 2.21.0
- Operating system: Arch Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description
Hello, 
as per title: In blame view, if you comment some line after the first line of a chain, the syntax highlighting is broken to the end of file. Tried with php and java files.
here are examples:
https://try.gitea.io/TbHG1AUy7x7vkj10/test-blame-bug/blame/branch/master/HelloWorld.java
https://try.gitea.io/TbHG1AUy7x7vkj10/test-blame-bug/blame/branch/master/index.php

## Screenshots
https://i.imgur.com/XcvSXpS.png
https://i.imgur.com/ISWfM76.png

**Comment by silverwind** [comment]
Your languages are dected correctly in those examples, which means this is a [highlight.js](https://github.com/highlightjs/highlight.js/) bug, report it there instead.

**Comment by eldiaboloz** [comment]
Thanks for pointing me in the right direction.

TLDR: there needs to be new lines in the html after every line of code for highlight.js to work correctly

After some fiddling with highlight.js and looking at the source 
and the comment regexes, and comparing the html source generated for the "normal" and "blame" views in gitea, there was a little difference: 
https://github.com/go-gitea/gitea/blob/master/routers/repo/blame.go#L239
https://github.com/go-gitea/gitea/blob/master/routers/repo/view.go#L313
so a simple change fixed the issue https://github.com/eldiaboloz/gitea/commit/bf39e6f475c96847bb39822999a3987d9c43461a

**Comment by lunny** [comment]
@eldiaboloz please send a PR

**Comment by eldiaboloz** [comment]
PR sent [6909](https://github.com/go-gitea/gitea/pull/6909)
